internal/commands/buildpack_new.go

```diff
@@ -104,7 +104,7 @@ func BuildpackNew(logger logging.Logger, creator BuildpackCreator) *cobra.Comman
 	cmd.Flags().StringSliceVarP(&flags.Stacks, "stacks", "s", nil, "Stack(s) this buildpack will be compatible with"+stringSliceHelp("stack"))
 	cmd.Flags().MarkDeprecated("stacks", "prefer `--targets` instead: https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md")
 	cmd.Flags().StringSliceVarP(&flags.Targets, "targets", "t", nil,
-		`Targets are the list platforms that one targeting, these are generated as part of scaffolding inside buildpack.toml file. one can provide target platforms in format [os][/arch][/variant]:[distroname@osversion@anotherversion];[distroname@osversion]
+		`Targets are the platforms that one targets. These are generated as part of scaffolding inside buildpack.toml file. One can provide target platforms in format [os][/arch][/variant]:[distroname@osversion@anotherversion];[distroname@osversion]
```

Comment:
@AidanDelaney , @buildpacks/toc or @jabrown85 
As native English speakers can you give me some :eyes: here?

Reply:
I think less is more here.  People will understand the intention from the examples.

```suggestion
		`Targets are of the form 'os/arch/variant', for example 'linux/amd64' or 'linux/arm64/v9'.  The full format for targets follows the form [os][/arch][/variant]:[distroname@osversion@anotherversion];[distroname@osversion]`
```

Reply:
I've applied the new text. I agree with 'less is more here' :-)

I've left the additional examples as they are. Not sure whether the intention was to get rid of those completley.